db-engine/src/main/java/com/splicemachine/db/iapi/types/DataType.java

```diff
@@ -1370,23 +1370,33 @@ public int compare(DataValueDescriptor o1, DataValueDescriptor o2) {
         }
     }
 
-    /**
-     *
-     * 256 gives < 1% normalized rank error
-     *
-     * @return
-     * @throws StandardException
+    /* 256 gives < 1% normalized rank error. Note that this error is independent to data
+     * stream size. Reference:
+     * https://datasketches.apache.org/docs/Quantiles/QuantilesAccuracy.html .
      */
     @Override
     public com.yahoo.sketches.quantiles.ItemsSketch getQuantilesSketch() throws StandardException {
         return com.yahoo.sketches.quantiles.ItemsSketch.getInstance(256,this);
     }
 
+    /* Bigger maxMapSize leads to better accuracy. Reference:
+     * https://datasketches.apache.org/docs/Frequency/FrequentItemsErrorTable.html .
+     * Memory consumption is (18 bytes * currentMapSize). currentMapSize starts from 8 and
+     * gets doubled when the number of items in map is bigger than 0.75 * currentMapSize.
+     * Also, currentMapSize is capped at maxMapSize.
+     */
     @Override
     public com.yahoo.sketches.frequencies.ItemsSketch getFrequenciesSketch() throws StandardException {
-        return new com.yahoo.sketches.frequencies.ItemsSketch(256);
+
+        return new com.yahoo.sketches.frequencies.ItemsSketch(8192);
```

Comment:
Maybe we can create a new static final field for this value?

Reply:
The idea is to make this parameter (together with other sketches' sizes) configurable. Jira: https://splicemachine.atlassian.net/browse/DB-11481 . So eventually we'd make the function accept a size argument from outside.